### PR TITLE
Random fixes

### PR DESCRIPTION
**Pull request by ret2libc**
These are fixes for things introduced in https://github.com/radare/radare2/commit/f90b534a4d0734f15f5d4a25b31a7e2e8c699ac4

### CODE REVIEW

File: libr/cons/cons.c [hunk 1]
```diff
@@ -1493,10 +1493,15 @@ R_API void r_cons_cmd_help(const char *help[], bool use_color) {
 				help[i], help[i + 1], help[i + 2], pal_reset);
 			continue;
 		}
-		// these are the normal lines
-		int padding = max_length - (strlen (help[i]) + strlen (help[i + 1]));
-		r_cons_printf ("| %s%s%s%*s  %s%s%s\n",
-			help[i], pal_args_color, help[i + 1],
-			padding, "", pal_help_color, help[i + 2], pal_reset);
+		if (!strcmp (help[i + 1], "") && !strcmp (help[i + 2], "")) {
```
Comment (radare) [inline libr/cons/cons.c]: Just check if the first char is nul, no need for strcmp

Comment (radare) [reply libr/cons/cons.c]: Also seems like u are chking 2 chars by using 2 strcmps

File: libr/cons/cons.c [hunk 2]
```diff
@@ -1493,10 +1493,15 @@ R_API void r_cons_cmd_help(const char *help[], bool use_color) {
...
+			// no need to indent the sections lines
+			r_cons_printf ("%s%s%s\n", pal_help_color, help[i], pal_reset);
+		} else {
+			// these are the normal lines
+			int padding = max_length - (strlen (help[i]) + strlen (help[i + 1]));
```
Comment (radare) [inline libr/cons/cons.c]: Can thid be negstive? Not sure how some libcs will behave with negative %*

**Comment by ret2libc** [comment]
appveyor is red just because master was red... I think the PR is ready.

**Pull request merged**